src/tribol/interface/mfem_tribol.hpp

```diff
@@ -238,16 +237,14 @@ std::unique_ptr<mfem::BlockOperator> getMfemBlockJacobian( integer csId );
 /**
  * @brief Returns gap vector to a given mfem::Vector
  *
- * @pre Coupling scheme cs_id must be registered using
- * registerMfemCouplingScheme()
- * @pre Redecomp mesh must be created and up to date by calling
- * updateMfemParallelDecomposition()
- * @pre Response vector must be up to date for current geometry by calling
- * update()
+ * @note This is stored as a dual vector, meaning the shared DOFs must be summed over all ranks to obtain their value.
```

Comment:
"This is stored as a mfem dual vector, meaning they expect the shared DOFs to be summed over all ranks to obtain the appropriate values."?